sklearn/manifold/spectral_embedding_.py

```diff
@@ -291,12 +295,14 @@ def spectral_embedding(adjacency, n_components=8, eigen_solver=None,
         laplacian = _set_diag(laplacian, 1, norm_laplacian)
         ml = smoothed_aggregation_solver(check_array(laplacian, 'csr'))
         M = ml.aspreconditioner()
+        # Create initial approximation X to eigenvectors
```

Comment:
I would also remove this comment

Reply:
Anyone trying to understand what is being done here would have to call up the LOBPCG documentation. If comments should generally be this sparse in sklearn then I can certainly remove it. But I honestly think the code could use the explanations.